query/branch/default_branch_all_build_steps_as_code.sql

```diff
@@ -1,10 +1,10 @@
 with repositories as (
   select
-    full_name
+    name_with_owner
```

Comment:
Does the previous and new column have the same values?

Reply:
Yup, `full_name` became `name_with_owner` on this table when we leaned into the naming.